### ISSUE TITLE
Moodle 4.1 - Cohorts managed by local_profilecohort retain gear icon

### ISSUE THREAD

**Issue by davidpesce** [moodle-an-hochschulen/moodle-local_profilecohort#50]
Just a heads up that with Moodle 4.1.x (and I'm guessing 4.0.x) the gear icon (Site admin > Users > Cohorts) still appears next to cohorts managed by local_profilecohort.

![Screenshot 2023-01-26 at 11 06 00 AM](https://user-images.githubusercontent.com/3988068/214887625-e8b1be8c-8084-42ea-bb2c-f310906c34b7.png)

I looked back at a 3.9 site and those icons do not appear.

**Comment by baumhof** [comment]
is the plugin working with moodle 4.1?

**Comment by abias** [comment]
Hi @davidpesce ,

many thanks for reporting this issue.
I could reproduce this finding and would agree that the managed cohorts should not have this cog icon / action menu.

To be able to understand if this is an issue of this plugin, I tested what happens with the latest release of https://moodle.org/plugins/local_cohortauto from Github. And managed cohorts from this plugin show a cog icon as well. So I think this is a Moodle core regression and not a problem of this plugin.

In Moodle tracker, I did not find a matching tracker issue for this regression.
So I created https://tracker.moodle.org/browse/MDL-78071. 
Unfortunately, I do not have the time to submit a fix there at the moment.

I will keep this issue here open as reference for admins who stumble upon the same issue in this plugin.

Cheers,
Alex
